src/js/components/covid19/Covid19Page.jsx

```diff
@@ -36,33 +33,90 @@ require('pages/covid19/index.scss');
 const propTypes = {
     loading: PropTypes.bool
 };
+const covid19Sections = [
+    {
+        section: 'overview',
+        label: 'Overview'
+    },
+    {
+        section: 'total_spending_by_budget_categories',
+        label: 'Total Spending by Budget Category'
+    },
+    {
+        section: 'award_spending_by_recipient',
+        label: 'Award Spending by Recipient'
+    },
+    {
+        section: 'award_spending_by_agency',
+        label: 'Award Spending by Sub-Agency'
+    },
+    {
+        section: 'award_spending_by_assistance_listing',
+        label: 'Award Spending by Assistance Listing'
+    },
+    {
+        section: 'data_sources_and_methodology',
+        label: 'Data Sources & Methodology'
+    },
+    {
+        section: 'other_resources',
+        label: 'Other Resources'
+    }
+];
 
 const Covid19Page = ({ loading }) => {
     const query = useQueryParams();
     const history = useHistory();
     const [activeSection, setActiveSection] = useState(query.section || 'overview');
     const dispatch = useDispatch();
     const { isRecipientMapLoaded } = useSelector((state) => state.covid19);
+    const jumpToSection = (section = '') => {
+        // we've been provided a section to jump to
+        // check if it's a valid section
+        const sectionObj = find(covid19Sections, ['section', section]);
+        if (!sectionObj) return;
 
-    const handleJumpToSection = (section) => {
-        jumpToSection(section);
+        // find the section in dom
+        const sectionDom = document.querySelector(`#covid19-${sectionObj.section}`);
+        if (!sectionDom) return;
 
         // add section to url
-        if (!window.location.href.includes(`section=${section}`)) {
-            history.replace(`${history.location.pathname}?section=${kebabCase(section)}`);
-        }
-        // update the state
+        history.replace(`?section=${sectionObj.section}`);
         setActiveSection(section);
-        Analytics.event({ category: 'COVID-19 - Profile', action: `${section} - click` });
```

Comment:
we need to keep analytics.events for a couple more sprints

Reply:
okay, added back in